x-pack/plugins/fleet/public/components/multiple_agent_policy_summary_line.tsx

```diff
@@ -0,0 +1,127 @@
+  const { getHref } = useLink();
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  // as default, show only the first policy
+  const policy = policies[0];
+  const { name, id } = policy;
+
+  const listItems: EuiListGroupItemProps[] = useMemo(() => {
+    return policies.map((p) => {
+      return {
+        'data-test-subj': `policy-${p.id}`,
+        label: p.name || p.id,
+        href: getHref('policy_details', { policyId: p.id }),
+        iconType: 'dot',
+        extraAction: {
+          color: 'text',
+          iconType: p.is_managed ? 'lock' : '',
+          alwaysShow: !!p.is_managed,
+          iconSize: 's',
+          'aria-label': 'Hosted agent policy',
+        },
+        showToolTip: !!p.is_managed,
+        toolTipText: i18n.translate('xpack.fleet.agentPolicySummaryLine.hostedPolicyTooltip', {
+          defaultMessage:
+            'This policy is managed outside of Fleet. Most actions related to this policy are unavailable.',
+        }),
+      };
+    });
+  }, [getHref, policies]);
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="xs">
+      <EuiFlexItem>
+        <EuiFlexGroup
+          direction={direction}
+          gutterSize={direction === 'column' ? 'none' : 's'}
+          alignItems="baseline"
+          style={MIN_WIDTH}
+          responsive={false}
+          justifyContent={'flexStart'}
+        >
+          <EuiFlexItem grow={false} className="eui-textTruncate">
+            <EuiFlexGroup style={MIN_WIDTH} gutterSize="s" alignItems="baseline" responsive={false}>
+              <EuiFlexItem grow={false} className="eui-textTruncate">
+                <EuiBadge color="default" data-test-subj="agentPolicyNameBadge">
+                  {name || id}
+                </EuiBadge>
+              </EuiFlexItem>
+              {policies.length > 1 && (
+                <EuiFlexItem grow={false}>
+                  <EuiBadge
+                    color="hollow"
+                    data-test-subj="agentPoliciesNumberBadge"
+                    onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+                    onClickAriaLabel="Open agent policies popover"
+                  >
+                    {`+${policies.length - 1}`}
+                  </EuiBadge>
+                  <EuiPopover
+                    data-test-subj="agentPoliciesPopover"
+                    isOpen={isPopoverOpen}
+                    closePopover={closePopover}
+                    anchorPosition="downCenter"
+                  >
+                    <EuiPopoverTitle>
+                      {i18n.translate('xpack.fleet.agentPolicySummaryLine.popover.title', {
+                        defaultMessage: 'This integration is shared by',
+                      })}
+                    </EuiPopoverTitle>
+                    <div style={{ width: '280px' }}>
+                      <EuiListGroup
+                        listItems={listItems}
```

Comment:
nit: is this the right component to use? comparing the designs vs the PR screenshots, seems like using this introduces a lot of extra spacing between the elements. how about using simple `<ul>` and `<li>`s instead?